cluster/kubemark/gce/config-default.sh

```diff
@@ -96,8 +96,7 @@ CONTROLLER_MANAGER_TEST_ARGS="${KUBEMARK_CONTROLLER_MANAGER_TEST_ARGS:-} ${CONTR
 SCHEDULER_TEST_ARGS="${KUBEMARK_SCHEDULER_TEST_ARGS:-} ${SCHEDULER_TEST_LOG_LEVEL} ${TEST_CLUSTER_API_CONTENT_TYPE} ${KUBEMARK_MASTER_COMPONENTS_QPS_LIMITS}"
 
 # Hollow-node components' test arguments.
-KUBELET_TEST_ARGS="--max-pods=100 $TEST_CLUSTER_LOG_LEVEL ${TEST_CLUSTER_API_CONTENT_TYPE}"
-KUBEPROXY_TEST_ARGS="${KUBEPROXY_TEST_LOG_LEVEL} ${TEST_CLUSTER_API_CONTENT_TYPE}"
+HOLLOW_PROXY_TEST_ARGS="${HOLLOW_PROXY_TEST_ARGS:-}"
```

Comment:
What about HOLLOW_KUBELET_TEST_ARGS?

Reply:
No usecase for it for now, but sure - I can add add.